openslides/agenda/static/templates/agenda/item-detail.html

```diff
@@ -22,20 +22,25 @@
           ng-click="item.project()"
           title="{{ 'Project item' | translate }}">
         <i class="fa fa-video-camera"></i>
+        {{ item.getContentResource().verboseName }}
       </a>
     </div>
     <h1>{{ item.getTitle() }}</h1>
-    <h2><translate>List of speakers</translate></h2>
+    <h2>
+      <translate>List of speakers</translate>
+      <span ng-if="item.speaker_list_closed" class="slimlabel label label-danger"
+          translate>Closed</span>
```

Comment:
No. That's exactly what I want. See also http://getbootstrap.com/components/#labels